routes/api-admin.php

```diff
@@ -49,3 +49,6 @@
 Route::post('/User::create', [UserController::class, 'create']);
 Route::post('/User::setEmail', [UserController::class, 'setEmail'])->withoutMiddleware([AdminCheck::class]);
 Route::post('/User::getEmail', [UserController::class, 'getEmail'])->withoutMiddleware([AdminCheck::class]);
+Route::post('/User::getCurrent', [UserController::class, 'getCurrent'])->withoutMiddleware([AdminCheck::class]);
+Route::post('/User::resetToken', [UserController::class, 'resetToken'])->withoutMiddleware([AdminCheck::class]);
+Route::post('/User::disableToken', [UserController::class, 'disableToken'])->withoutMiddleware([AdminCheck::class]);
```

Comment:
```suggestion
Route::post('/User::deleteToken', [UserController::class, 'deleteToken'])->withoutMiddleware([AdminCheck::class]);
```

maybe?

Or

```suggestion
Route::post('/User::unsetToken', [UserController::class, 'unsetToken'])->withoutMiddleware([AdminCheck::class]);
```

maybe? `unset` matches `reset`.

If there is a method `disableX` I would also expect to have a method `enabledX` which we don't.